service-tests/jetbrains.js

```diff
@@ -3,31 +3,53 @@
 const Joi = require('joi');
 const ServiceTester = require('./runner/service-tester');
 const { isMetric } = require('./helpers/validators');
+const { isSemver } = require('./helpers/validators');
 
-const t = new ServiceTester({ id: 'jetbrains', title: 'JetBrains plugin' });
+const t = new ServiceTester({ id: 'jetbrains', title: 'JetBrains plugin/version' });
 module.exports = t;
 
 t.create('downloads (number as a plugin id)')
   .get('/plugin/d/7495.json')
   .expectJSONTypes(Joi.object().keys({ name: 'downloads', value: isMetric }));
 
+t.create('version (number as a plugin id)')
+  .get('/plugin/v/7495.json')
+  .expectJSONTypes(Joi.object().keys({ name: 'jetbrains plugin', value: isSemver }));
+
```

Comment:
Thanks for all the great tests! Instead of interspersing these with the download tests, could you keep all the download tests together followed by all the version tests? They're a lot easier to follow that way.

Reply:
Resorted.